appveyor.yml

```diff
@@ -52,7 +52,8 @@ install:
   # - choco install -y imdisk-toolkit
   # - mkdir tmp
   # - imdisk -a -t file -m tmp -o awe -s 1G -p "/fs:ntfs /q /y"
-  - set GROONGA_INSTALL_SUB_FOLDER=インストール
+  # - set GROONGA_INSTALL_SUB_FOLDER=インストール
```

Comment:
Is this comment need?
I think that no problem to remove this comment.

Reply:
It is better to test japanese directory name, but not succeeded yet.(because of AppVeyor specific reason, maybe)
So, for the record, I intentionally keep it left as a comment.